src/bin/diff.js

```diff
@@ -2,41 +2,110 @@
 
 'use strict'
 
+var path = require('path')
+var argv = require('yargs')
+  .boolean('verbose')
+  .alias('v', 'verbose')
+  .argv
+
 var size = require('window-size')
 var availableWidth = size.width || /* istanbul ignore next */ 80
 var ui = require('cliui')({width: availableWidth})
 
 var getRuleFinder = require('../lib/rule-finder')
-var difference = require('../lib/array-diff')
+var arrayDifference = require('../lib/array-diff')
+var objectDifference = require('../lib/object-diff')
 var getSortedRules = require('../lib/sort-rules')
 
-var rules = getSortedRules(
-  difference(
-    getRuleFinder(process.argv[2]).getCurrentRules(),
-    getRuleFinder(process.argv[3]).getCurrentRules()
-  )
-)
-
-var outputRules, outputRuleCellMapper
 var outputPadding = ' '
 var outputMaxWidth = 0
 var outputMaxCols = 0
 
-/* istanbul ignore next */
-if (rules.length) {
-  console.log('\n diff rules\n') // eslint-disable-line no-console
-  rules = rules.map(function columnSpecification(rule) {
-    rule = rule + outputPadding
-    outputMaxWidth = Math.max(rule.length, outputMaxWidth)
-    return rule
+var files = [argv._[0], argv._[1]]
+var collectedRules = (
+  argv.verbose ? [files] : [files, [].concat(files).reverse()]
+).map(
```

Comment:
Sure, will rearrange it and possibly make some comments. I fear otherwise, it would give us a hard time understanding it comprehensively in some weeks 😜